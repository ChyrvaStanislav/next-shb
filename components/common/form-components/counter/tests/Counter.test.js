import React from 'react';
import { render } from '@testing-library/react';
import Counter from '../index';

const Component = () => <Counter value="25" maxLength={150} className="someClass" />;

describe('Counter renders', () => {
  it('should Counter renders', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
