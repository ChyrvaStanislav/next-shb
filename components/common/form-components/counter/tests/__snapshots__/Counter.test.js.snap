// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter renders should Counter renders 1`] = `
<DocumentFragment>
  <span
    class="counter someClass"
  >
    2/150
  </span>
</DocumentFragment>
`;
